docs/DeveloperGuide.md

```diff
@@ -390,13 +390,36 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     
         Use case resumes at step 2. 
 
+**Use case: Clear past interactions**
+
+**MSS**
+1.  User enters the clear command.
+2.  StonksBook clears the GUI.
```

Comment:
I think that clears the GUI might not be clear enough as the the contact list is also a GUI. You could consider chatbot GUI or something that it is clearer?

Reply:
or clear the chat messages in the chatbot GUI

Reply:
Yeah I'll do a quick fix.